lib/SILOptimizer/Utils/InstOptUtils.cpp

```diff
@@ -864,8 +864,8 @@ void swift::placeFuncRef(ApplyInst *ai, DominanceInfo *domInfo) {
 /// Add an argument, \p val, to the branch-edge that is pointing into
 /// block \p Dest. Return a new instruction and do not erase the old
 /// instruction.
-TermInst *swift::addArgumentToBranch(SILValue val, SILBasicBlock *dest,
-                                     TermInst *branch) {
+TermInst *swift::addArgumentsToBranch(SmallVectorImpl<SILValue> &vals,
```

Comment:
Since `vals` is not modified, use an `ArrayRef<SILValue>` here instead.

Reply:
Fixed!